.ebextensions/cronjob.config

```diff
@@ -286,6 +286,27 @@ files:
         content: |
           */5 * * * * root aws cloudwatch put-metric-data --metric-name ProcessCount --namespace GoodJob --value=$(./current/script/good_job_number_of_processes.sh) --unit Count --region='eu-west-2' --dimensions InstanceId=$(ec2-metadata --instance-id | cut -d " " -f 2)
 
+    #
+    # Send Review Group Tariffs Reminders
+    #
+    "/etc/cron.d/send_review_group_tariffs_reminders":
+        mode: "000644"
+        owner: root
+        group: root
+        content: |
+          0 0 1 * * root /bin/bash -l -c 'cd /var/app/current && RAILS_ENV=production bin/run_as_webapp bundle exec rake school_groups:send_review_group_tariffs_reminder >> log/jobs.log 2>&1'
```

Comment:
I don't think this needs to run every day and better to send the emails later in the day I think rather than at midnight.

You could further simplify the code in the services further by doing all the scheduling in the cronjob, e.g. "0 8 15 3,9 *" will schedule the cronjob to run at 8am on the 15th of March and September.